docs/configure

```diff
@@ -478,10 +480,11 @@ ditch_maxmsgsize:
 		and domain = '${quote_mysql:$domain}' \
 		and users.maxmsgsize > 0 \
 		and users.domain_id=domains.domain_id}{${value}K}fail} Kb.
-  local_part_suffix = -*
+  local_part_suffix = +*
```

Comment:
What about using an `.ifdef  ... .endif` like I suggested in a previous comment? This would make toggling this feature globally much easier.